pytorch_transformers/tokenization_utils.py

```diff
@@ -519,14 +519,30 @@ def _convert_token_to_id_with_added_voc(self, token):
     def _convert_token_to_id(self, token):
         raise NotImplementedError
 
-
-    def encode(self, text):
+    def encode(self, text, text_pair=None, add_special_tokens=False):
```

Comment:
I like this interface; one concern is that `add_special_tokens` needs to be True for RoBERTa. In the case of a sentence pair it's fine, because you return a tuple, but for a single sentence it will silently produces wrong output in this case:
```python
input = tokenizer.encode('Hello world')
roberta(torch.tensor([input]))
```

Any thoughts on how to fail more explicitly in this case? Some possibilities with various tradeoffs:
- make this function always return a tuple, with either 1 or 2 elements (may break other assumptions)
- default add_special_tokens to True (not great from an interface design perspective)
- check for the beginning of sentence index (=0) in the roberta forward (may add a little extra overhead, kinda hacky)

Reply:
As a temporary measure and until we think of something better, I'm adding a check for the beginning of the sentence index (=0) in the RoBERTa forward

If it is not found, it sends a warning: `A sequence with no special tokens has been passed to the RoBERTa model. This model requires special tokens in order to work.Please specify add_special_tokens=True in your encoding.`